packages/media/src/seekVideo.js

```diff
@@ -29,19 +29,15 @@ function seekVideo(video, offset = 0.99) {
   }
 
   return new Promise((resolve, reject) => {
-    // The "suspend" event can be fired multiple times, even before the browser
-    // has started seeking in the first place. Because of that, it's not
-    // desired to reject on "suspend" straight away.
-    // Instead, delay rejection in the hopes that the "canplay" event fires
-    // in the meantime.
-    video.addEventListener('suspend', (evt) => {
+    // If the seek takes longer 3 seconds, guess it timed out and error out.
+    video.addEventListener('seeking', (evt) => {
       const wait = setTimeout(() => {
         clearTimeout(wait);
         reject(evt);
       }, THREE_SECONDS);
```

Comment:
@swissspidy why was this 3 seconds in the first place. Now it based on seeking, I would if this could fail quickly.

Reply:
The inline comment explained it.

The suspend event is potentially fired multiple times, even if the browser eventually loads the video.

It rejects the error if the video was „properly“ suspended and the browser really never loaded it. In my testing 3s was a reasonable time to wait.

What you have now is listening to the seeking event, which is immediately fired when the browser starts seeking, to start the timeout.
That seems odd to me. With this, I don‘t think it makes sense to set the timeout in there. Have you tried moving that out of there? No need to listen for seeking at all.

Reply:
Wasn't the question I asked here, why was 3 seconds? Just a random number?

Should I just remove this?

Reply:
> In my testing 3s was a reasonable time to wait.

Reply:
The `suspend` listener was added because `canplay` potentially never fired. Are you certain that `seeked` is always fired? I am a bit skeptical.

Need to ensure there's proper handling for when seeking doesn't happen.

You mentioned `stalled` and `aborted` on Slack as events to potential listen for.

Maybe you need to give that a try.

Reply:
Seeked event would always fire on success. See [docs](https://html.spec.whatwg.org/multipage/media.html#event-media-seeked). 

Hooking into seeking, makes sense as once the seek start, there is then 3 seconds for it to be fulfilled or it will be rejected.